README.md

```diff
@@ -1,3 +1,43 @@
-# near-abi-rs
+<div align="center">
 
-NEAR smart contract ABI primitives
+  <h1><code>near-abi-rs</code></h1>
+
+  <p>
+    <strong>Rust library providing NEAR ABI models.</strong>
+  </p>
+
+  <p>
+    <a href="https://github.com/near/near-abi-rs/actions/workflows/test.yml?query=branch%3Amain"><img src="https://github.com/near/near-abi-rs/actions/workflows/test.yml/badge.svg" alt="Github CI Build" /></a>
+    <a href="https://crates.io/crates/near-abi"><img src="https://img.shields.io/crates/v/near-abi.svg?style=flat-square" alt="Crates.io version" /></a>
+    <a href="https://crates.io/crates/near-abi"><img src="https://img.shields.io/crates/d/near-abi.svg?style=flat-square" alt="Download" /></a>
+    <a href="https://docs.rs/near-abi"><img src="https://docs.rs/near-abi/badge.svg" alt="Reference Documentation" /></a>
+  </p>
+
+  <h3>
+      <a href="https://github.com/near/abi">ABI</a>
+      <span> | </span>
+      <a href="https://docs.rs/near-abi">Reference Documentation</a>
+      <span> | </span>
+      <a href="#contributing">Contributing</a>
+  </h3>
+</div>
+
+## Release notes
+
+**Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
+
+## Overview
+
+❗ **Warning: ABI is still in early stages of development so expect breaking changes to this library until we reach 1.0**
+
+This library is meant to serve as an unopinionated source of Rust models for [ABI](https://github.com/near/abi).
+
+## ABI Metaschema
+
+This repo also contains meta [JSON Schemas](https://json-schema.org/) of ABI. These schemas can be found in the [`metaschema`](/metaschema) folder: `near-abi-${version}-schema.json` for a specific ABI schema version or `near-abi-current-schema.json` for what is currently in the `main` branch of the repository.
```

Comment:
Broadly speaking, what do you think about also hosting the meta schemas of ABI in a more generic repo like https://github.com/near/abi, instead of the `abi-rs` one?

Reply:
Yes, I think it is sensible to port finalized metaschemas to a more visible repo. This repo can just serve as our automated source of truth.